.rubocop.yml

```diff
@@ -48,6 +45,9 @@ Style/WordArray:
 Style/ClassAndModuleChildren:
     Enabled: false
 
+Style/SymbolArray:
+  Enabled: false
```

Comment:
and we have we disabled this?

Reply:
If this was left it would complain about the array of event modes see: https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/SymbolArray however I couldn't get the `%I` stuff working

Reply:
This is what you need it to be to get it to pass that lint:

`enum event_mode: %i[draft prerelease live ended]`

Reply:
Done!